cfme/intelligence/chargeback/rates.py

```diff
@@ -0,0 +1,259 @@
+
+
+class EditStorageChargebackView(EditComputeChargebackView):
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and
+            self.title.text == 'Storage Chargeback Rate "{}"'.format(self.obj.description))
+
+
+class ComputeRate(Updateable, Pretty, Navigatable):
+    pretty_attrs = ['description']
+
+    def __init__(self, description=None,
+                 currency=None,
+                 fields=None,
+                 appliance=None,
+                 ):
+        Navigatable.__init__(self, appliance=appliance)
+        self.description = description
+        self.currency = currency
+        self.fields = fields
+
+    def __getitem__(self, name):
+        return self.fields.get(name)
+
+    def create(self):
+        view = navigate_to(self, 'New')
+        view.fill_with({'description': self.description,
+                        'currency': self.currency,
+                        'fields': self.fields},
+                       on_change=view.add_button,
+                       no_change=view.cancel_button)
+
+        view.flash.assert_success_message('Chargeback Rate "{}" was added'.format(
+            self.description))
+
+    def update(self, updates):
+        view = navigate_to(self, 'Edit')
+        view.fill_with(updates, on_change=view.save_button, no_change=view.cancel_button)
+        view.flash.assert_success_message('Chargeback Rate "{}" was saved'.format(
+            updates.get('description')))
+
+    def delete(self):
+        view = navigate_to(self, 'Details')
+        view.configuration.item_select('Remove from the VMDB', handle_alert=True)
+        view.flash.assert_success_message('Chargeback Rate "{}": Delete successful'.format(
+            self.description))
+
+
+@navigator.register(ComputeRate, 'All')
+class ComputeRateAll(CFMENavigateStep):
+    VIEW = RatesView
+    prerequisite = NavigateToAttribute('appliance.server', 'IntelChargeback')
+
+    def step(self):
+        self.view.rates.tree.click_path(
+            "Rates",
+            "Compute"
+        )
+
+
+@navigator.register(ComputeRate, 'New')
+class ComputeRateNew(CFMENavigateStep):
+    VIEW = AddComputeChargebackView
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        self.view.configuration.item_select("Add a new Chargeback Rate")
+
+
+@navigator.register(ComputeRate, 'Details')
+class ComputeRateDetails(CFMENavigateStep):
+    VIEW = RatesDetailView
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        self.view.rates.tree.click_path(
+            "Rates",
+            "Compute", self.obj.description
+        )
+
+
+@navigator.register(ComputeRate, 'Edit')
+class ComputeRateEdit(CFMENavigateStep):
+    VIEW = EditComputeChargebackView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.configuration.item_select("Edit this Chargeback Rate")
+
+
+class StorageRate(ComputeRate, Updateable, Pretty, Navigatable):
```

Comment:
You can inherit just the ComputeRate, all the other three are already inherited :)